Dockerfile

```diff
@@ -1,2 +1,63 @@
-FROM kylemanna/openvpn
+FROM kylemanna/openvpn:latest
 
+ARG K8S_VERSION=v1.3.6
+
+RUN wget https://storage.googleapis.com/kubernetes-release/release/$K8S_VERSION/bin/linux/amd64/kubectl
+RUN chmod +x kubectl
+RUN mv kubectl /usr/local/bin/
+
+ARG GITHUB_TOKEN
+ARG REPO=cloudposse/github-pam
+ARG FILE=github-pam_linux_386
+ARG VERSION=0.10
+ARG PAM_SCRIPT_VERSION=1.1.8-1
+
+ADD rootfs /
+
+RUN if [ ! -z $GITHUB_TOKEN ]; then \
+      set -ex \
+      && echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories \
+      && apk update \
+      && apk add --no-cache --virtual .build-deps \
+		    curl \
+		    jq \
+		    linux-pam \
+		    pamtester \
+		  && gh-dl-release $VERSION github-pam-plugin \
+      && chmod +x github-pam-plugin \
+      && mv github-pam-plugin /bin/ \
+      && apk add ca-certificates \
+      && apk del .build-deps; \
+    else \
+      echo '`GITHUB_TOKEN` required for fetching github-pam-plugin'; \
+      exit 1; \
+    fi
+
+RUN set -ex \
+      && apk update \
+      && apk add linux-pam \
+      && apk add --no-cache --virtual .build-deps \
```

Comment:
Yea, not seen that before. Alpine is definitely catering to the docker generation.